lib/block-supports/border.php

```diff
@@ -6,26 +6,36 @@
  */
 
 /**
- * Registers the style attribute used by the border feature if needed for block types that
- * support borders.
+ * Registers the style attribute used by the border feature if needed for block
+ * types that support borders.
  *
  * @param WP_Block_Type $block_type Block Type.
  */
 function gutenberg_register_border_support( $block_type ) {
-	// Determine border related features supported.
-	// Border width, style etc can be added in the future.
-	$has_border_radius_support = gutenberg_block_has_support( $block_type, array( '__experimentalBorder', 'radius' ), false );
+	// Determine if any border related features are supported.
+	$has_border_color_support = gutenberg_has_border_support( $block_type, 'color' );
```

Comment:
Can we favor `gutenberg_block_has_support` over `gutenberg_has_border_support` (remove this one)?

Reply:
I can make that change, `gutenberg_block_has_support` didn't exist when I first created this and I didn't notice it after the rebase. Thanks for bringing it to my attention.

Reply:
✅ Done.